src/DataAccess/DoctrineDonationRepository.php

```diff
@@ -60,7 +60,7 @@ private function newDonationEntity( Donation $donation ): DoctrineDonation {
 		} else {
 			$doctrineDonation->setCity( $donation->getPersonalInfo()->getPhysicalAddress()->getCity() );
 			$doctrineDonation->setEmail( $donation->getPersonalInfo()->getEmailAddress() );
-			$doctrineDonation->setName( $donation->determineFullName() );
+			$doctrineDonation->setName( $donation->getPersonalInfo()->getPersonName()->getFullName() );
```

Comment:
The capitalization is correct in "Anonym". This field is used as the author of a comment and should start with an uppercase letter.